gravitee-am-management-api/gravitee-am-management-api-standalone/gravitee-am-management-api-standalone-container/src/main/java/io/gravitee/am/management/standalone/spring/StandaloneConfiguration.java

```diff
@@ -88,11 +89,13 @@ public Node node() {
     }
 
     @Bean
+    @Qualifier("commonEventManager")
     public EventManager eventManager() {
         return new EventManagerImpl();
     }
 
     @Bean
+    @Qualifier("managementObjectMapper")
```

Comment:
Remove this redundant "@Qualifier" annotation and rely on the @Bean method.

Reply:
> Remove this redundant "@qualifier" annotation and rely on the @bean method.

I has been introduced to avoid a bean injection exception. This exception has been fixed by fixing and upgrading the cockpit connetor https://github.com/gravitee-io/gravitee-cockpit-connectors/pull/221. I forgot to remove this qualifier :+1: